src/blockchain/web3_extentions/direct_deposit.py

```diff
@@ -0,0 +1,42 @@
+import logging
+
+import variables
+from blockchain.contracts.staking_module import StakingModuleContract
+from blockchain.typings import Web3
+from metrics.metrics import MELLOW_VAULT_BALANCE
+
+logger = logging.getLogger(__name__)
+
+
+def is_mellow_depositable(
+    w3: Web3,
+    module_id: int
+) -> bool:
+    try:
```

Comment:
Let's place in try only calls to contracts

Reply:
```suggestion
   # check if mellow enabled

    try:
        module_id = w3.lido.simple_dvt_staking_strategy.staking_module_contract.get_staking_module_id()
        balance = w3.lido.simple_dvt_staking_strategy.vault_balance()
     except:
         ....
         return False
         
     # rest logic with checks
```

Reply:
Done